pkg/asset/installconfig/powervs/session.go

```diff
@@ -90,28 +102,161 @@ func getPISession() (*ibmpisession.IBMPISession, *UserCredentials, error) {
 			},
 		}, &apikey)
 		if err != nil {
-			return nil, nil, errors.New("Error saving the API_KEY variable")
+			return nil, errors.New("Error saving the API_KEY variable")
 		}
 	}
 
-	region = os.Getenv("IBMCLOUD_REGION")
+	uc := &UserCredentials{UserID: id, APIKey: apikey}
+
+	return uc, err
+}
+
+/*
+//  https://github.com/IBM-Cloud/power-go-client/blob/master/ibmpisession/ibmpowersession.go
+*/
+func getPISession(region string, zone string, uc *UserCredentials) (*ibmpisession.IBMPISession, error) {
+
+	// @TOOD: query if region is multi-zone? or just pass through err...
+	// @TODO: pass through debug?
+	s, err := ibmpisession.New(uc.APIKey, region, false, defSessionTimeout, uc.UserID, zone)
+	return s, err
+}
+
+// getRegionInfo   returns information usually collected by platform.go but is required here
+// 		   because ibmpisession requires this information as part of session.go
+//		   this is found here because session.go is responsible for collecting
+//		   os.Getenv variables, and therefore isnt found in region.go
+func getRegionInfo() (string, string, error) {
+	var region, sessionRegion, zone, sessionZone string
+	var err error
+
+	// -------------------------
+	// Region
+	// -------------------------
+
+	// Retrieve sessionRegion from the current enviornment context
+	sessionRegion = os.Getenv("IBMCLOUD_REGION")
 	// this can also be pulled from  ~/bluemix/config.json
 	if r2 := os.Getenv("IC_REGION"); len(r2) > 0 {
 		if len(region) > 0 && region != r2 {
-			return nil, nil, errors.New(fmt.Sprintf("conflicting values for IBM Cloud Region: IBMCLOUD_REGION: %s and IC_REGION: %s", region, r2))
+			return "", "", errors.New(fmt.Sprintf("conflicting values for IBM Cloud Region: IBMCLOUD_REGION: %s and IC_REGION: %s", region, r2))
```

Comment:
i think you'd mentioned earlier that the code for the API keys doesn't check for conflicts? So it probably makes sense just to have one override other and not return an error here (to be consistent). IOW -- just delete this check.

Reply:
That comes down in pull request 51.  I'd actually prefer to solve this by making all of these use the same method, getEnv, and to add a bit of functionality to getEnv, or wrap getEnv... because all of these variables are handled the same way, and require survey prompts, and error checks... I figure we could drop that all into one function we can just call.  I figure we could add support for checking files for the same information at that time too.

Basically, 
findStuff( ["VarName, VarName2", "Survey Prompt", "Survey Description", "FileName" )
Check if any of the permutations of the variable name have been exported with os.Getenv
If not, check if any of the permutations of the variable name exist in the provided file name (if provided)
If not, prompt the user with the survey prompt, and description

But this change was out of scope for this PR so I just moved this over.

Reply:
Sure, if we can find a way to streamline these later that's fine. But okay with deleting this check for now?